examples/user_info_custom_provider/Rocket.toml.example

```diff
@@ -0,0 +1,5 @@
+[global.oauth.github]
+provider = { auth_uri = "https://github.com/login/oauth/authorize", token_uri = "https://github.com/login/oauth/access_token" }
```

Comment:
Edited the `auth_uri` and restarted Rocket to confirm it was getting picked up from `Rocket.toml` rather than the existing `providers!` macro.